src/Tools/AI Test Toolkit/src/Logs/AITLogEntry.Table.al

```diff
@@ -0,0 +1,231 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.TestTools.AITestToolkit;
+
+using System.Reflection;
+using System.TestTools.TestRunner;
+
+table 149034 "AIT Log Entry"
+{
+    DataClassification = SystemMetadata;
+    DrillDownPageId = "AIT Log Entries";
+    Extensible = false;
+    Access = Internal;
+    ReplicateData = false;
+
+    fields
+    {
+        field(1; "Entry No."; Integer)
+        {
+            Caption = 'Entry No.';
+            AutoIncrement = true;
+        }
+        field(2; "AIT Code"; Code[100])
+        {
+            Caption = 'AIT Code';
+            NotBlank = true;
+            TableRelation = "AIT Test Suite";
+        }
+        field(3; "AIT Line No."; Integer)
+        {
+            Caption = 'Line No.';
+        }
+        field(4; "Start Time"; DateTime)
+        {
+            Caption = 'Start Time';
+        }
+        field(5; "End Time"; DateTime)
+        {
+            Caption = 'End Time';
+        }
+        field(6; "Message"; text[250])
+        {
+            Caption = 'Message';
+        }
+        field(7; "Codeunit ID"; Integer)
+        {
+            Caption = 'Codeunit ID';
+        }
+        field(8; "Codeunit Name"; Text[250])
+        {
+            Caption = 'Codeunit Name';
+            FieldClass = FlowField;
+            CalcFormula = lookup(AllObjWithCaption."Object Caption" where("Object Type" = const(Codeunit), "Object ID" = field("Codeunit ID")));
+        }
+        field(9; "Duration (ms)"; integer)
+        {
+            Caption = 'DurationInMs (ms)';
+        }
+        field(10; Status; Option)
+        {
+            Caption = 'Status';
+            OptionMembers = Success,Error;
+        }
+        field(11; Operation; Text[100])
+        {
+            Caption = 'Operation';
+        }
+        field(13; Version; Integer)
+        {
+            DataClassification = SystemMetadata;
+            Caption = 'Version';
+        }
+        field(15; Tag; Text[20])
+        {
+            DataClassification = CustomerContent;
+            Caption = 'Tag';
+        }
+        field(16; "Error Call Stack"; Text[2048]) //TODO: Consider changing this to blob
+        {
+            DataClassification = CustomerContent;
+            Caption = 'Error Call Stack';
+        }
+        field(17; "Procedure Name"; Text[128])
+        {
+            Caption = 'Procedure Name';
+            DataClassification = CustomerContent;
+        }
+        field(18; RunID; Guid)
+        {
+            DataClassification = SystemMetadata;
+            Caption = 'RunID';
+        }
+        field(20; "Orig. Operation"; Text[100])
+        {
+            Caption = 'Orig. Operation';
+        }
+        /// <summary>
```

Comment:
What about putting these into the `Description` property? Or are there any plans to obsolete the property in favor of field comments?